### PR TITLE
Updating readme.txt for 2.7 Release

### PR DESCRIPTION
**Pull request by jimtrue**
Provides Read Me Update/Change Log for #4455

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -177,8 +178,45 @@ Join us in further translating the Pods interface at: [https://translate.wordpre
...
+This upgrade requires a minimum PHP version of 5.3+ and WordPress 4.4+.
+
 == Changelog ==
 
+= 2.7 - December 4th 2017 =
+## New PHP & WP Minimum Version Requirements:
+- PHP Version Change Requirement 5.3+
+- WP Version Change Required 4.4+
+
+## Major Changes in 2.7 Release: Flexible Relationships:
```
Comment (quasel) [inline readme.txt]: Maybe add link to https://www.youtube.com/watch?v=1h_5tQ6LmoY as it's a really well done Video?

File: readme.txt [hunk 2]
```diff
@@ -177,8 +178,45 @@ Join us in further translating the Pods interface at: [https://translate.wordpre
...
+- **Dynamic Field Views**: The relationship fields and file upload fields have been rewritten to take advantage of "Dynamic Field Views", driven by JavaScript. More technical details about the directions we went towards these field changes can be read about in our ["Pods 2.7: What We Did and Why We Did It"](**Needs URL**)
+
+## Significant Improvements and Fixes:
+- Enhancements for Date, Time & Date/Time Fields to improve class inheritance, use WP Defaults and allow for custom options for saving and display @JoryHOgeveen
+- Gallery & Tile View for Upload/Media Fields #3589 @JoryHogeveen 
+- IsEmpty for all Date/Time Fields, Number, Currency, etc. improvements for `[if field][/if]` for Templates @JoryHogeveen 
+- **Major fixes for Pods Templates**: Proper handling of shortcodes within Pods Templates and in if/each blocks. Proper handling of all image tags and user traversal @jamesgol 
+- Comment Traversal #4118 @sc0ttkclark
+- Update Compatibility with Admin Columns 3.0 #4570 @JoryHogeveen
```
Comment (quasel) [inline readme.txt]: I think that affects Admin Columns Pro too? (current pro Version is 4.1)

File: readme.txt [hunk 3]
```diff
@@ -177,8 +178,45 @@ Join us in further translating the Pods interface at: [https://translate.wordpre
 
 We also have a dedicated [Slack Chat](https://pods.io/chat/) channel to help our translators get started and to support them on the process.
 
+== Upgrade Notice ==
+
+= 2.7 =
+This upgrade requires a minimum PHP version of 5.3+ and WordPress 4.4+.
```
Comment (sc0ttkclark) [inline readme.txt]: WP 4.5+

File: readme.txt [hunk 4]
```diff
@@ -177,8 +178,45 @@ Join us in further translating the Pods interface at: [https://translate.wordpre
...
+
 == Changelog ==
 
+= 2.7 - December 4th 2017 =
+## New PHP & WP Minimum Version Requirements:
+- PHP Version Change Requirement 5.3+
+- WP Version Change Required 4.4+
```
Comment (sc0ttkclark) [inline readme.txt]: WP 4.5+

File: readme.txt (hunk 1 above)
Comment (sc0ttkclark) [reply readme.txt]: Done below

**Pull request merged**